.travis.yml

```diff
@@ -0,0 +1,38 @@
+sudo: required
+
+services:
+  - docker
+
+matrix:
+  fast_finish: true
+
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
```

Comment:
ah, good question. This was something I found as an accepted answer on stackoverflow and I hadn't bothered to try without that option. 

The reason I found this was because I discovered that travisci was using an older version of docker-engine. I never noticed in other projects but it was apparent here due to the use of the `--build-arg` flag which was not a supported flag in the version of docker-engine that is there by default. Can we run this command without the options set here, maybe, I haven't tried.